database/buildSchema/40_reviewer_action_list.sql

```diff
@@ -37,6 +38,11 @@ CREATE FUNCTION review_list (stageid int, reviewerid int)
             AND (review = NULL
             OR is_locked = FALSE)) != 0 THEN
             'SELF_ASSIGN'
+        WHEN COUNT(*) FILTER (WHERE (appstatus = 'CHANGES_REQUIRED'
+            OR appstatus = 'DRAFT')
+            AND review_assignment.status = 'ASSIGNED'
+            AND review_status_history.status = 'SUBMITTED') != 0 THEN
+            'AWAITING_RESPONSE'
```

Comment:
Your editor had messed up the whitespace formatting in the sql files. Have pushed a fix so I can see what's actually changed: 1dbb13ff

Can you check your settings so we keep this consistent?

Reply:
Ah, I don't think it was my editor. I did this change because the spacing looked odd to me. But yes, you would have to set the option "Remove space changes" to see the diff using Github